core/src/main/java/org/apache/calcite/sql/SqlWindow.java

```diff
@@ -610,8 +610,10 @@ public boolean isAllowPartial() {
       if (orderList.size() > 0) {
         // if order by is a compound list then range not allowed
         if (orderList.size() > 1 && !isRows()) {
-          throw validator.newValidationError(isRows,
-              RESOURCE.compoundOrderByProhibitsRange());
+          if (!onlyLiteralBounds(lowerBound, upperBound)) {
```

Comment:
Can you add a sql case to the misc.iq file to test the results？

Reply:
Added test case